src/XPurchase/VirtoCommerce.XPurchase/Schemas/CartType.cs

```diff
@@ -41,8 +41,8 @@ public CartType(ICartAvailMethodsService cartAvailMethods, IDynamicPropertyResol
             Field<MoneyType>("total", resolve: context => context.Source.Cart.Total.ToMoney(context.Source.Currency));
             Field<MoneyType>("subTotal", resolve: context => context.Source.Cart.SubTotal.ToMoney(context.Source.Currency));
             Field<MoneyType>("subTotalWithTax", resolve: context => context.Source.Cart.SubTotalWithTax.ToMoney(context.Source.Currency));
-            Field<MoneyType>("extendedPriceTotal", resolve: context => context.Source.Cart.Items.Sum(i => i.ExtendedPrice).ToMoney(context.Source.Currency));
-            Field<MoneyType>("extendedPriceTotalWithTax", resolve: context => context.Source.Cart.Items.Sum(i => i.ExtendedPriceWithTax).ToMoney(context.Source.Currency));
+            Field<MoneyType>("extendedPriceTotal", resolve: context => context.Source.Cart.Items.Where(x => !x.IsGift).Sum(i => i.ExtendedPrice).ToMoney(context.Source.Currency));
```

Comment:
Could you separate Gifts with normal line items on an aggregate level? In order to avoid such in-line logic